src/gui/editors/TimeLineWidget.cpp

```diff
@@ -455,4 +457,15 @@ void TimeLineWidget::contextMenuEvent(QContextMenuEvent* event)
 	menu.exec(event->globalPos());
 }
 
+
+TimeLineWidget::AutoScrollState TimeLineWidget::getDefaultAutoScrollState()
```

Comment:
We omit the `get` prefix. Qt does the same so I suppose we also do it for consistency.